giftcompare/giftcompare/permissions.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import permissions
+
+class IsAdminOrReadOnly(permissions.BasePermission):
+    """
+    Custom permission to only allow admin users to edit it.
+    """
+    def has_permission(self, request, view):
+        if request.method in permissions.SAFE_METHODS:
+            # The method is a safe method
+            return True
+        else:
+            # The method isn't a safe method
+            # Only allow it if the user is an admin
+            return request.user.is_staff
+        
+class IsAdminOrNothing (permissions.BasePermission):
```

Comment:
I think you could have done this with the built-in 'IsAdminUser' instead of creating a new one but if it works, ok.

Reply:
Cool, thanks for letting me know, I replaced the custom with the built-in 'IsAdminUser'

Reply:
Cool - thanks!